src/GitHubCreateMergePRs/config.xml

```diff
@@ -28,9 +28,11 @@
 
     <!-- Roslyn feature branches -->
     <merge from="master" to="features/static-lambdas" owners="jcouv" />
-    <merge from="master" to="features/patterns3" owners="gafter" frequency="weekly" />
+    <merge from="master" to="features/covariant-returns" owners="gafter" />
+    <merge from="master" to="features/target-typing" owners="gafter" />
```

Comment:
This may result in merges being attempted once every 3 hours. If this is too often to be manageable for you, you may want to add `frequency="daily"` or `frequency="weekly"`.

Reply:
I'm not worried about the frequency.  It has the advantage that if one blocks, a later one merging will close all the earlier ones. Which will make it easier to keep up to date without much work.

Reply:
Confirmed with @gafter that this was fine.